UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/HomeController.cs

```diff
@@ -21,10 +21,11 @@ public IActionResult Index()
             return View();
         }
 
-        [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+        [Route("/error")]
         public IActionResult Error()
         {
```

Comment:
Please add unit test cases for Home Controller as well.

Reply:
Changes done.

Reply:
Added unit test for home controller